foreman/foreman.spec

```diff
@@ -183,8 +183,8 @@ BuildRequires: %{?scl_prefix}rubygem(activerecord-session_store) >= 0.1.1
 BuildRequires: %{?scl_prefix}rubygem(activerecord-session_store) < 2
 BuildRequires: %{?scl_prefix_ror}rubygem(sprockets) >= 3
 BuildRequires: %{?scl_prefix_ror}rubygem(sprockets) < 4
-BuildRequires: %{?scl_prefix_ror}rubygem(sprockets-rails) >= 2.3.3
-BuildRequires: %{?scl_prefix_ror}rubygem(sprockets-rails) < 3
+BuildRequires: %{?scl_prefix}rubygem(sprockets-rails) >= 3
+BuildRequires: %{?scl_prefix}rubygem(sprockets-rails) < 4
```

Comment:
We have to build this again in `tfm` SCL as  ror42 does not contain this package.